src/components/progress-circle/progress-circle.ts

```diff
@@ -50,6 +50,20 @@ export class MdProgressCircle implements OnDestroy {
   private _interdeterminateInterval: number;
 
   /** @internal */
+  get ariaValueMin() {
+    if (this.mode != 'indeterminate') {
```

Comment:
Add comment explaining why we want `valuemin` and `valueman` to be omited for indeterminate.